sdk/storage/azure-storage-extensions/setup.py

```diff
@@ -6,6 +6,18 @@
 
 import os
 from setuptools import setup, Extension
+from wheel.bdist_wheel import bdist_wheel
+
+
+class bdist_wheel_abi3(bdist_wheel):
```

Comment:
This looks pretty hacky, but it comes from a recommendation from [cibuildwheel docs](https://cibuildwheel.readthedocs.io/en/stable/faq/#abi3).

Reply:
Maybe, but it's definitely not a global thing, so I'm not side-eyeing it.

Not only that, but this isn't even something that CAN be future proofed, as `setup.py` will STILL BE MANDATORY for extension packages.